pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -2801,17 +3083,40 @@ def DisposalCapacityRule(model, k, t):
     )
 
     def TreatmentCapacityExpansionRule(model, r):
-        return model.v_T_Capacity[r] == sum(
-            (
-                model.p_sigma_Treatment[r, wt]
-                * sum(model.vb_y_Treatment[r, wt, j] for j in model.s_J)
-                + sum(
-                    model.p_delta_Treatment[wt, j] * model.vb_y_Treatment[r, wt, j]
-                    for j in model.s_J
+        if model.config.objective == Objectives.cost_surrogate:
+            return (
+                model.v_T_Capacity[r]
+                == sum(
+                    (
+                        model.p_sigma_Treatment[r, wt]
+                        * sum(model.vb_y_Treatment[r, wt, j] for j in model.s_J)
+                        + sum(
+                            model.p_delta_Treatment[wt, j]
+                            * model.vb_y_Treatment[r, wt, j]
+                            for j in model.s_J
+                        )
+                    )
+                    for wt in model.s_WT
+                )
+                + pyunits.convert_value(
+                    100000,
```

Comment:
We require adjustments to set an upper bound on surrogate flow. Note that we already have upper and lower 'cap' bounds, which may allow us to deactivate this constraint for surrogates if we're utilizing the cap upper bound. Additionally, let's incorporate user estimates for setting the upper bound (desalination plant capacity). It’s also crucial to verify the accuracy of surrogate extrapolation, considering the bounds used during model training

Reply:
As discussed I changed this to be the upper bound of flow on which the surrogate was trained on.